README.md

```diff
@@ -2,25 +2,46 @@ easy-license-creator
 ====================
 [![Build Status](https://travis-ci.org/DANS-KNAW/easy-license-creator.png?branch=master)](https://travis-ci.org/DANS-KNAW/easy-license-creator)
 
-<Remove this comment and extend the descriptions below>
-
+Create a license (pdf file) for a given dataset.
 
 SYNOPSIS
 --------
 
-    easy-license-creator params
+    easy-license-creator <datasetID> <template-dir> <license-file>
 
 
 DESCRIPTION
 -----------
 
-<Replace with a longer description of this module>
+A command line tool to create a license for a given dataset as a pdf file. The tool searches for a dataset that corresponds to the given `datasetID` and uses the metadata of this dataset, as well as the personal data of the depositor to generate the license.
+
+The License Creator uses a template of the license with placeholders in which the data is substituted. After all the placeholders are filled with actual data, it is converted into a PDF file.
```

Comment:
had to read it twice: `in which the data is substituted` perhaps:

The License Creator uses a template with placeholders. After replacing the placeholders with actual data, the template is converted into a PDF file.